diagnostic.md

```diff
@@ -51,13 +61,18 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request DELETE http://www.example.com/donuts/$ID
 ```
```

Comment:
This won't work without setting the $ID variable.

`curl --include --request DELETE http://www.example.com/donuts/1
`